### DISCUSSION TITLE
Unable to run 2 spark Jobs parallelly

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#33] [Q&A] [answered]
Hi Team, 

Scenario: We are running a notebook that triggers a spark job while simultaneously loading tables to Lakehouse from the Lakehouse file section, which triggers another job. However, we are unable to run these jobs in parallel. We are using an F4 SKU in one tenant, but in another tenant with the same configuration, we can run two jobs in parallel. Could you please provide insights into what might be causing this issue?

Thank you

**Comment by ramizsha** [comment, marked as answer]
Hi @SanchanaMohankumar,
The discussions here are about the development proccess of a workload, this is a general Fabric question, please contact Fabric support.
